workshops/lookups.py

```diff
@@ -96,8 +96,23 @@ class AirportLookup(ModelLookup):
     )
 
 
+class LanguageLookup(ModelLookup):
+    model = models.Language
+    search_fields = (
+        'name__icontains',
+        'subtag__icontains',
+    )
+
+    def get_query(self, request, term):
```

Comment:
We can sort the first 10 most popular languages alphabetically while the rest are listed by their frequency.